src/java/org/apache/cassandra/config/Config.java

```diff
@@ -515,6 +515,11 @@
      */
     public volatile int validation_preview_purge_head_start_in_sec = 60 * 60;
 
+    // Using String instead of ConsistencyLevel here to keep static initialization from cascading and starting
+    // threads during tool usage mode. See CASSANDRA-12988 and DatabaseDescriptorRefTest for details
+    public String auth_read_consistency_level = "LOCAL_QUORUM";
+    public String auth_write_consistency_level = "EACH_QUORUM";
```

Comment:
As those variables can be accessed and modify by multiple threads they should be `volatile`.